hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/UserAccountCreationFailedState.java

```diff
@@ -22,11 +21,14 @@ public UserAccountCreationFailedState(
         SessionId sessionId,
         boolean transactionSupportsEidas) {
 
-        super(requestId, authnRequestIssuerEntityId, sessionExpiryTimestamp, assertionConsumerServiceUri, sessionId, transactionSupportsEidas);
+        super(requestId, authnRequestIssuerEntityId, sessionExpiryTimestamp, assertionConsumerServiceUri, relayState, sessionId, transactionSupportsEidas);
 
         this.relayState = relayState;
     }
 
+    // Keep this for now to make deserialization with the previous version of UserAccountCreationFailedState compatible
```

Comment:
Same question as before. Why oh why?

Reply:
The parent class has this property already